_includes/footer.html

```diff
@@ -2,7 +2,6 @@
   <div>
     <div class="main">
       <div class="logo"><img src="{{ "/images/logo.svg" | relative_url }}" /></div>
-      <span class="md-hidden">The open source multicloud control plane</span>
```

Comment:
should this be changed to "Manage your applications and infrastructure the Kubernetes way" instead of being removed?

Reply:
@jbw976 per Matt, we are simply removing this text from the footer